backend/controllers/brandController.js

```diff
@@ -4,22 +4,14 @@ const bcrypt = require("bcrypt"); // For password hashing
 // Signup a brand
 exports.signup = async (req, res) => {
   const { name, email, password, category } = req.body;
-  const existingBrand = await Brand.findOne({ email});
```

Comment:
if we remove this, how are we gonna check for the unique emailId for the brands?

Reply:
I had added this in the validation. so it is no needed here